### DISCUSSION TITLE
landscape fullscreen on mobile

### DISCUSSION THREAD

**Discussion by chibykiss** [sampotts/plyr#2356] [Q&A]
i cant seem to get a landscape full screen on mobile devices. the full screen on mobile is only portrait. pls is this how the fullscreen  works on a default, or is there a way to activate the landscape full screen on mobile.

**Comment by 0xcrypto** [comment]
@chibykiss use events with the native api

```javascript
player.on('enterfullscreen', event => {
    screen.orientation.lock('landscape');
});

player.on('exitfullscreen', event => {
    screen.orientation.lock('portrait');
});
```
